tests/test-backend-ops.cpp

```diff
@@ -82,7 +82,6 @@ static std::vector<float> tensor_to_float(const ggml_tensor * t) {
 
     ggml_type_traits_t tt = ggml_internal_get_type_traits(t->type);
     size_t bs = ggml_blck_size(t->type);
-    std::vector<float> vq(ggml_blck_size(t->type));
```

Comment:
The one in the inner loop should be removed instead, this was intended to remove the allocations from the inner loop.

Reply:
Got it, let's see if the current solution will pass the CI or if an additional cleanup for that vector is required